test/integration/basic/pages/custom-extension.jsx

```diff
@@ -0,0 +1,4 @@
+import {World} from '../components/world'
```

Comment:
Add another case where we are importing with `../components/world.jsx`

Reply:
Done 👍